src/trace_processor/perfetto_sql/stdlib/trace/stats.sql

```diff
@@ -0,0 +1,87 @@
+--
+-- Copyright 2025 The Android Open Source Project
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     https://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+-- Reports the duration of the flush operation for cloned traces (for each
+-- buffer).
+CREATE PERFETTO TABLE clone_flush_latency (
```

Comment:
Can we please add a prefix this with traced? This is part of the presubmit checks in tools/run_presubmit we are missing.

Reply:
Good idea, done